src/Compatibility/Microsoft.DotNet.ApiSymbolExtensions/Filtering/DocIdSymbolFilter.cs

```diff
@@ -21,12 +25,15 @@ public class DocIdSymbolFilter(string[] docIdsToExcludeFiles) : ISymbolFilter
         public bool Include(ISymbol symbol)
         {
             string? docId = symbol.GetDocumentationCommentId();
-            if (docId is not null && _docIdsToExclude.Contains(docId))
+            //if there's no doc id, always include it
+            if (docId is null) return true;
+            //if the doc id is in the list, return the meaning of the list
+            if (_docIds.Contains(docId))
```

Comment:
The old tool had some special handling for members, https://github.com/dotnet/arcade/blob/c80e079caaf7e892889651abd089f7569243f0f4/src/Microsoft.Cci.Extensions/Filters/DocIdIncludeListFilter.cs#L43-L44
Do you think we need that here, or is it reasonable to tell folks to also list the type if they list a member?

Reply:
Good question. I don't have a strong opinion. For our purposes, it is easy to provide the type, but I could see value in the type being implied by the member. At some point, implicit rules become hard to reason about, but this one seems straightforward.